tasks/stakingV2.ts

```diff
@@ -0,0 +1,213 @@
+import { subtask, task, types } from "hardhat/config"
+import { StakedTokenBPT__factory, StakedTokenMTA__factory, StakedToken__factory } from "types/generated"
+import { simpleToExactAmount } from "@utils/math"
+import { formatUnits } from "@ethersproject/units"
+import { getSigner } from "./utils/signerFactory"
+import { logTxDetails } from "./utils/deploy-utils"
+import { getChain, resolveAddress } from "./utils/networkAddressFactory"
+
+subtask("staked-stake", "Stake MTA or mBPT in V2 Staking Token")
```

Comment:
Subtasks can be called form other tasks, but not from test files, right?

Reply:
Yes, subtasks can be called from tasks. eg https://github.com/mstable/mStable-contracts/blob/b4fbb024b617344d0d1671f31d76a8025320d360/tasks/weekly.ts#L38 

I think if we want a test to run something in a task we move the logic into a function. We've done that a number of times already